From ae9225d55963c4420c49ccfa3f2fafc416f92032 Mon Sep 17 00:00:00 2001
From: Andreas Schwab <schwab@linux-m68k.org>
Date: Mon, 24 Aug 2026 18:00:26 +0200
Subject: [PATCH] m68k: remove sysdeps/m68k/m680x0/fpu/w_fmod_compat.c (bug
 34559)

This was missed from commit 6deadd4eb6.

(cherry picked from commit ff1c5580610dee6d743169fc0f6684505d372dc9)
---
 sysdeps/m68k/m680x0/fpu/w_fmod_compat.c | 15 ---------------
 1 file changed, 15 deletions(-)
 delete mode 100644 sysdeps/m68k/m680x0/fpu/w_fmod_compat.c

diff --git a/sysdeps/m68k/m680x0/fpu/w_fmod_compat.c b/sysdeps/m68k/m680x0/fpu/w_fmod_compat.c
deleted file mode 100644
index 57f38091e6..0000000000
--- a/sysdeps/m68k/m680x0/fpu/w_fmod_compat.c
+++ /dev/null
@@ -1,15 +0,0 @@
-/* m68k provides an optimized __ieee752_fmod.  */
-#include <math-svid-compat.h>
-#ifdef SHARED
-# undef SHLIB_COMPAT
-# define SHLIB_COMPAT(a, b, c) 1
-# undef LIBM_SVID_COMPAT
-# define LIBM_SVID_COMPAT 1
-# undef compat_symbol
-# define compat_symbol(a, b, c, d)
-#include <math/w_fmod_compat.c>
-libm_alias_double (__fmod_compat, fmod)
-#else
-#include <math-type-macros-double.h>
-#include <w_fmod_template.c>
-#endif
